schema/GeneticVariant_Alt_ID_Database_properties.mcf

```diff
@@ -0,0 +1,1401 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema.py
+
+Node: dcid:fanconi_Anaemia_Mutation_Database_(FANCM)_ID
```

Comment:
could we remove '(' and ')' from property names?

Reply:
removed